models/action.js

```diff
@@ -15,6 +15,7 @@ const ActionSchema = new Schema({
   user_id: String,
   field: String, // Used when an action references a particular field of an object. (e.g. a flag on a username or bio)
   detail: String, // Describes the reason for an action (e.g. 'Username is offensive')
+  description: String, // Describes the reason for the action in more detail (e.g. A description of why the comment is offensive.)
```

Comment:
It would make more sense to me to use words like:

```
field -> metadata.field // Describes the field for which the action is related to.
detail -> metadata.reason // Describes the reason that the action was taken (if any)
description -> metadata.comment // Describes with more words why the action was taken in the user's own words
```

Essentially moving these pieces into a new `metadata` field.